packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/CalendarHeader.tsx

```diff
@@ -15,7 +16,9 @@ export function Header({ onClose }: TimePickerCalendarProps) {
 
   return (
     <div className={styles.container}>
-      <TimePickerTitle>Select a time range</TimePickerTitle>
+      <TimePickerTitle>
+        <Trans i18nKey="time-picker.calendar-select-range.title">Select a time range</Trans>
```

Comment:
@joshhunt according to the structure `area.component.message`, should this key be `time-picker.calendar-header.select-time`? 🤔

Reply:
hmmm, seeing how basic this component is, i think it's more like `time-picker.calendar.title`?